src/components/AddFreakModal/AddFreakModal.jsx

```diff
@@ -1,19 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import { Button } from 'react-bootstrap';
 import Modal from '../Modal/Modal';
 import EditFreakForm from '../EditFreakForm/EditFreakForm';
+import { FreakModelDefault } from '../../models/freaks';
 
 function AddFreakModal({ title, isOpen, onClose }) {
+  const [ freak, setFreak ] = useState(FreakModelDefault);
+
   return (
     <Modal
       title={ title }
       isOpen={ isOpen }
       headerContent={ <button className="filter__button-reset" type="button">Reset</button> }
-      footerContent={ <Button className="filter__button-apply" variant="primary py-2 px-5 ">Submit</Button> }
+      footerContent={ <Button className="filter__button-apply" variant="primary py-2 px-5 " onClick={ setFreak }>Apply</Button> }
```

Comment:
This onClick is not doing anything meaninful. it is even populating the `freak` state with the event object. Better to remove